x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/kibana_deprecation_errors.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut } from '@elastic/eui';
+
+interface Props {
+  errorType: 'pluginError' | 'requestError';
+}
+
+const i18nTexts = {
+  pluginError: i18n.translate('xpack.upgradeAssistant.kibanaDeprecationErrors.pluginErrorMessage', {
+    defaultMessage:
+      'Not all Kibana deprecations were retrieved successfully. This list may be incomplete.',
```

Comment:
This may occur if the core deprecations service was unable to fetch a deprecation for a particular plugin. I've updated the error message to mention to check the Kibana logs in this scenario.